src/Combat.cpp

```diff
@@ -0,0 +1,27 @@
+#include "Combat.h"
+
+void CombatSystem::on_tick()
+{
+    for(auto&& [attack_entity, attack, attack_pos, sprite, flip] : QueryAttacks :: access_storage().each()){
```

Comment:
All the `{` go in their own line, make it more readable where the blocks start and end